node-js-basics-101/node_modules/asynquence-contrib/plugin.wrap.js

```diff
@@ -0,0 +1,92 @@
+// "wrap"
+ASQ.wrap = function $$wrap(fn,opts) {
+	function checkThis(t,o) {
+		return (!t ||
+			(typeof window != "undefined" && t === window) ||
+			(typeof global != "undefined" && t === global)
+		) ? o : t;
+	}
+
+	function paramSpread(gen) {
+		return function *paramSpread(token) {
+			yield *gen.apply(this,token.messages);
```

Comment:
__[jshint]__
* 'yield' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->